Grant necessary Kibana application privileges to `reporting_user` role (#118058)
elastic/elasticsearch: docs/reference/security/authorization/built-in-roles.asciidoc

Grants the minimum privileges required to collect monitoring data for the {stack}.

[[built-in-roles-reporting-user]] `reporting_user`::
Grants the necessary privileges required to use {reporting} features in {kib},
including generating and downloading reports. This role implicitly grants access

> to the reporting indices, with each user having access only to their own reports.

Reviewer: ```suggestion
to all Kibana reporting features, with each user having access only to their own reports.
```
